src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -793,17 +825,106 @@ def load_module(name, value):
         # import it here to avoid circular import
         from diffusers import pipelines
 
-        # 6. Load each module in the pipeline
+        ###### 6. device map delegation ######
```

Comment:
can we create a method for this? I think we also plan to support other device mapping strategies in the future, and it will be a little bit too overwhelming